robotd/master.py

```diff
@@ -11,12 +11,23 @@
 import pyudev
 import setproctitle
 
-from robotd.devices import BOARDS
+import textwrap
+
+from robotd.devices import BOARDS, STARTUP_BOARDS
+
+
+def _send(connection, message):
+    # Split the message into buf_size chunks
+    buf_size = BoardRunner.SOCK_BUFFER_SIZE
+    for msg in [message[i:i + buf_size] for i in range(0, len(message), buf_size)]:
+        connection.send(msg)
```

Comment:
I stand corrected.

Given those semantics I'm not clear on why we need to use multiple send calls?